src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,56 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int INDEX_OF_NAME = 0;
+    private static final int INDEX_OF_NUM = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder builder = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String data = reader.readLine();
+            while (data != null) {
+                builder.append(data).append(" ");
+                data = reader.readLine();
+            }
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException("Can't find file" + e);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file" + e);
+        }
+
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(toFileName))) {
+            String allData = getReport(builder.toString());
+            writer.write(allData);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write data to file" + e);
+        }
+    }
 
+    private String getReport(String allData) {
+        StringBuilder builder1 = new StringBuilder();
+        String [] arr = allData.split(" ");
```

Comment:
Don't use abbreviations and acronyms in variable names. The name must reflect the contents of the variable. 
check it in other places